src/connector/postgres.rs

```diff
@@ -486,7 +486,11 @@ impl PostgresUrl {
 
         if let Some(connect_timeout) = self.query_params.connect_timeout {
             config.connect_timeout(connect_timeout);
-        };
+        }
+
+        if let Some(schema) = &self.query_params.schema {
+            config.search_path(schema.clone());
+        }
```

Comment:
I see we move things out of query_params just above. Could we avoid the clone here?

Reply:
Things above can be moved because they implement `Copy`. Whether it's `Duration`, `SslMode`, `ChannelBinding` or `bool`

Reply:
Ach, I would have expected that to be in a `From` impl or otherwise take self by value, but indeed, it's not the case.